docs/development-ide-configuration.md

```diff
@@ -1,22 +1,22 @@
 # IDE Configuration for developing UDS Packages
 
-This document explains how to configure specific IDEs to improve your development experience when developing UDS packages, such as setting up schema validation for Zarf packages, UDS bundles and Maru tasks. 
+This document explains how to configure specific IDEs to improve your development experience when developing UDS packages, such as setting up schema validation for Zarf packages, UDS bundles and Maru tasks.
 
 ## VSCode
 
-To configure your VSCode for schema validation, you must install a yaml language support [extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml). From there you have three options. You can configure VSCode globally, on a per repo basis or per file basis to provide validations. 
+To configure your VSCode for schema validation, you must install a yaml language support [extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml). From there you have three options. You can configure VSCode globally, on a per repo basis or per file basis to provide validations.
 
 ### Global / Workspace Config
```

Comment:
Consider redirecting to external documentation, we probably don't want to get in the game of documenting other products.

Reply:
out of scope for this PR, got caught in a delete trailing whitespace command I ran

Reply:
def out of scope of @corang's PR. FWIW we agreed to do this when we removed the .vscode/settings.json and `# yaml-language-server: $schema=https://raw.githubusercontent.com/defenseunicorns/zarf/main/zarf.schema.json` from all of the packages and the zarf.yamls because we should not be assuming that people are using vscode.